package.json

```diff
@@ -32,7 +32,11 @@
                     ".story",
                     ".steps"
                 ],
-                "configuration": "./language-configuration.json"
+                "configuration": "./language-configuration.json",
+                "icon": {
+                    "dark": "./images/icon.png",
```

Comment:
Do you have an example?

Reply:
<img width="334" alt="image" src="https://user-images.githubusercontent.com/30923336/210341411-1afcded3-654b-4549-b1ab-a6763c4fffa0.png">
recently share it in out chat :)